code/modules/mob/living/simple_animal/hostile/deer.dm

```diff
@@ -0,0 +1,155 @@
+/*
+TODO -
+Have them evaluate mobs in their vicinity
+	If it's a deer, or disguised as a deer, do nothing
+	If it's a hostile mob, run away
+	If it's a carbon mob, evaluate
+		If the carbon mob has run intent on, higher chance of running away
+		If the carbon mob has deer clothing on, lower chance of running away
+
+	If a gunshot is heard within their vicinity, run away //CANT DO
+*/
+/mob/living/simple_animal/hostile/deer
+	name = "deer"
+	desc = "Doe. A deer. A female Deer."
+	icon_state = "deer"
+	icon_living = "deer"
+	icon_dead = "deer_dead"
+	faction = "deer"
+	health = 50
+	maxHealth = 50
+	size = SIZE_NORMAL
+	response_help  = "pets"
+
+	attacktext = "kicks"
+	melee_damage_lower = 5
+	melee_damage_upper = 15
+
+	minbodytemp = 200
+
+/mob/living/simple_animal/hostile/deer/GiveTarget(var/new_target)
+	if(isDead())
+		return //The deers problems are over now, try not to be too jealous
+	if(isliving(new_target))
+		if(is_spooked(new_target))
+			var/list/can_see = view(src, vision_range)
+			get_spooked(new_target)
+			for(var/mob/living/simple_animal/hostile/deer/D in can_see)
+				D.get_spooked(new_target)
+
+/mob/living/simple_animal/hostile/deer/proc/is_spooked(var/mob/living/target)
+	if(friends.Find(target))
+		return 0
+	if(ishuman(target))
+	/*
+	Making it so that there is some tactics to hunting deer
+		If you're running at them like a crazy man, expect for them to get spooked
+		Otherwise, if you're wearing clothing made of deer, or are holding an apple, they are less likely to get spooked
+	*/
+		var/mob/living/carbon/human/H = target
+		var/spook_prob = 30
+
+		if(H.m_intent == "run") //Don't run while stalking deer
+			spook_prob += 70
+		if(H.wear_suit)
+			if(istype(H.wear_suit, /obj/item/clothing/suit/leather/deer))
+				visible_message("Has leather suit of deer on")
+				spook_prob -= 10
+		if(H.head)
```

Comment:
Do if(istype(H:head, /obj/item/clothing/head/leather/deer)) as it's a bit odd referencing these